plugins/package-managers/gradle-model/src/main/kotlin/Extensions.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2021 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.plugins.packagemanagers.gradlemodel
+
+import OrtDependency
```

Comment:
Is there a reason why the classes within `GradleModel.kt` are not within any package?

And or, could all be put into the same package, e.g. `org.ossreviewtoolkit.plugins.packagemanagers.gradlemodel` ?

Reply:
> Is there a reason why the classes within `GradleModel.kt` are not within any package?

Yes, this is a requirement for use with the Gradle Tooling API (TAPI). I don't recall the details, but I remember there were some problem with using FQN for classes that are sent across the model builder API.